DQMOffline/Trigger/python/SusyMonitoring_cff.py

```diff
@@ -157,6 +157,32 @@
 double_soft_muon_backup_90_mhtpt.numGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_DoubleMu3_DZ_PFMET90_PFMHT90_v*')
 double_soft_muon_backup_90_mhtpt.denGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_v*')
 
+#triple muon
+triple_muon_mupt = hltTOPmonitoring.clone()
+triple_muon_mupt.FolderName   = cms.string('HLT/SUSY/SOS/TrileMu/Muon')
+# Selections
+triple_muon_mupt.nmuons           = cms.uint32(3)
+triple_muon_mupt.invMassUppercut       = cms.double(50)
+triple_muon_mupt.invMassLowercut       = cms.double(10)
+triple_muon_mupt.invMassCutInAllMuPairs=cms.bool(True)
+# Binning
+
+# Triggers
+triple_muon_mupt.numGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_TripleMu_5_3_3_Mass3p8to60_DZ_v*')
+triple_muon_mupt.denGenericTriggerEventPSet.hltPaths = cms.vstring('HLT_Trimuon2_Upsilon5_Muon_v*')
```

Comment:
can you please check the name of this trigger path ?
HLT_Trimuon2_Upsilon5_Muon_v

in addition, it would be useful to see some (filled) plots produced by your code ...